Documentation/network/bgp-control-plane.rst

```diff
@@ -326,6 +327,13 @@ Cilium Agent's logs rather than a more specific error message.
 
 .. _RFC-2385 : https://www.rfc-editor.org/rfc/rfc2385.html
 
+If a ``CiliumBGPPeeringPolicy`` is deployed with an ``authSecretRef`` that cannot be 
+found, the BGP session will be using an empty password  and an error such as the following will be logged:
+
+::
```

Comment:
```suggestion
If a ``CiliumBGPPeeringPolicy`` is deployed with an ``authSecretRef`` that Cilium cannot find, the BGP session will use an empty password and the agent will log an error such as in the following example::
```

Not sure we need to specify the error itself, it's probably enough to mention there will be an error (the error message is descriptive enough that people should be able to match it with the case you describe), but I'm not opposed to it if you prefer to keep it like this.